veggieseasons/lib/widgets/veggie_headline.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:veggieseasons/data/veggie.dart';
+import 'package:veggieseasons/styles.dart';
+
+class VeggieHeadline extends StatelessWidget {
+  final Veggie veggie;
+
+  const VeggieHeadline(this.veggie);
+
+  List<Widget> _buildSeasonDots(List<Season> seasons) {
+    List<Widget> widgets = <Widget>[];
+
+    for (Season season in seasons) {
+      widgets.add(SizedBox(width: 4.0));
+      widgets.add(
+        Container(
+          height: 10.0,
+          width: 10.0,
+          decoration: BoxDecoration(
+            color: Styles.seasonColors[season],
+            borderRadius: BorderRadius.circular(5.0),
+          ),
+        ),
+      );
+    }
+
+    return widgets;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        /* navigation forthcoming */
```

Comment:
[nit] Prepending `TODO(redbrogdon):` to the comment here will make it easier for the rest of us to make you feel bad later.

Reply:
😆